pkg/ddevapp/amplitude_project.go

```diff
@@ -56,14 +56,14 @@ func (app *DdevApp) TrackProject() {
 		PhpVersion(app.GetPhpVersion()).
 		ProjectType(app.GetType()).
 		RouterDisabled(IsRouterDisabled(app)).
-		TraefikEnabled(globalconfig.DdevGlobalConfig.UseTraefik).
 		WebserverType(app.GetWebserverType())
 
 	if !nodeps.ArrayContainsString(containersOmitted, "db") {
 		builder.
 			DatabaseType(app.Database.Type).
 			DatabaseVersion(app.Database.Version)
 	}
+	builder.Router(globalconfig.DdevGlobalConfig.Router)
```

Comment:
```suggestion

	if !IsRouterDisabled(app) {
		builder.Router(globalconfig.DdevGlobalConfig.Router)
	}
```

For optional properties we should use a condition or make them required.

Reply:
Changed in pushed code, thanks.